### PR TITLE
Add UI for empty state for bridge when wallet is not connected

### PR DESCRIPTION
**Pull request by barryhalo**
## Description of Changes
Add empty state for bridge when wallet is not connected
Display

Screen Shot 2021-09-07 at 10 15 15 PM

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDEV-236)

### Developer Checklist:

* [x] I have followed the guidelines in our Contributing document
* [x] This PR has a corresponding JIRA ticket
* [x] My branch conforms with our naming convention i.e. `feature/HDF-XXX-description`
* [x] All files have appropriate file headers and documentation
* [ ] I have written new tests for your core changes, as applicable
* [ ] I have successfully ran tests locally

### Reviewers Checklist:
* [x] Code is readable and understandable; any unclear parts have explanations 
* [x] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

### CODE REVIEW

File: src/pages/Tailwind/Bridge/index.tsx [hunk 1]
```diff
@@ -19,11 +21,28 @@ const NotSupportedContent = () => {
...
+}
+
 const CurrentPanelContent = () => {
   const { chainId } = useActiveWeb3React()
   const features = NETWORK_SUPPORTED_FEATURES[chainId as ChainId]
-  if (features?.bridge) {
+
+  if (features?.bridge && chainId !== 1) {
     return <BridgePanel />
+  } else if (chainId === 1) {
```
Comment (0xAplki) [inline src/pages/Tailwind/Bridge/index.tsx]: just checking why we need to check if `chainId` is 1? We still want main net right? Works well in polygon tho based from the deployed link.

**Review by 0xAplki — COMMENTED** [review]
Just left a comment. Working when I'm in polygon. Mainnet does not work though. Not sure if this is the desired behavior. I also re-ran the test seems to be passing now :)

File: src/pages/Tailwind/Bridge/index.tsx (hunk 1 above)
Comment (schystz) [reply src/pages/Tailwind/Bridge/index.tsx]: I think a better condition is `else if (!account)` (add `account` to the destructure vars on line 39)

Currently the connect button always show up when selected network is Ethereum, even though wallet is already connected

File: src/pages/Tailwind/Bridge/index.tsx [hunk 2]
```diff
@@ -19,11 +21,28 @@ const NotSupportedContent = () => {
     </div>
   )
 }
+
+const NotConnected = () => {
+  const toggleWalletModal = useWalletModalToggle()
+  const caption = 'Get access to the bridge right connect to you wallet!'
```
Comment (chris247474) [inline src/pages/Tailwind/Bridge/index.tsx]:
```suggestion
  const caption = 'Connect your wallet to start bridging your tokens!'
```

**Review by chris247474 — CHANGES_REQUESTED** [review]
@barryhalo i think needs a bit more testing, doesn't yet detect when the wallet is loaded 

See video here https://halodao-project.slack.com/archives/C025Q3SUE81/p1631088917140800?thread_ts=1631064332.128500&cid=C025Q3SUE81

image

**Comment by barryhalo** [comment]
> @barryhalo i think needs a bit more testing, doesn't yet detect when the wallet is loaded
> 
> See video here https://halodao-project.slack.com/archives/C025Q3SUE81/p1631088917140800?thread_ts=1631064332.128500&cid=C025Q3SUE81
> 
> image

@ctverceles fixed now

**Comment by barryhalo** [comment]
@schystz @0xAplki ready for review

File: src/pages/Tailwind/Bridge/index.tsx [hunk 3]
```diff
@@ -19,11 +21,28 @@ const NotSupportedContent = () => {
...
+
 const CurrentPanelContent = () => {
-  const { chainId } = useActiveWeb3React()
+  const { chainId, account } = useActiveWeb3React()
   const features = NETWORK_SUPPORTED_FEATURES[chainId as ChainId]
-  if (features?.bridge) {
+
+  if (features?.bridge && account !== null) {
     return <BridgePanel />
+  } else if (account === null) {
```
Comment (schystz) [inline src/pages/Tailwind/Bridge/index.tsx]: @barryhalo `account` can also be undefined, so a simpler condition would be `else if (!account)` which should cover both null & undefined

**Review by 0xAplki — COMMENTED** [review]
Working for me! tried a variety of switches on other networks as well. Same comment as @schystz on the code but I'll go on and do the first tick. :)

**Review by 0xAplki — APPROVED** [review]

**Comment by chris247474** [comment]
All working for me! Re-requested review but consider this a tick (guess its just James' comemnt thats left then we can merge this)!

File: src/pages/Tailwind/Bridge/index.tsx (hunk 3 above)
Comment (barryhalo) [reply src/pages/Tailwind/Bridge/index.tsx]: @schystz copy!

**Comment by barryhalo** [comment]
@schystz for review, falsey !account value

**Review by schystz — APPROVED** [review]
Looks good now @barryhalo !

**Pull request merged**